tests/alive-tv/params/func-fail.src.ll

```diff
@@ -0,0 +1,13 @@
+; TEST-ARGS: -func=b -func=c
+
+define i32 @a() {
+  ret i32 0
+}
+
+define i32 @b() {
+  ret i32 1
+}
+
+define i32 @c() {
+  ret i32 2
```

Comment:
this test was supposed to fail. Is it failing? if yes, it needs the ERROR: thing

Reply:
Yes, I added the missing ERROR check.
BTW, I realized that the lit testing isn't checking absence of ERROR when dealing with src.ll/tgt.ll with multiple functions. Maybe I can look into this.